rpcs3/Emu/Cell/lv2/sys_ss.cpp

```diff
@@ -300,18 +311,40 @@ error_code sys_ss_update_manager(u64 pkg_id, u64 a1, u64 a2, u64 a3, u64 a4, u64
 			// *i think* this gives english
 			vm::write8(a2, a1 == 0x48C1B ? 0x1 : 0);
 		}
+		break;
 	}
-	else if (pkg_id == 0x600C)
+	case 0x600C:
 	{
 		// write eeprom
+		break;
 	}
-	else if (pkg_id == 0x6009)
+	case 0x6009:
 	{
 		// get seed token
+		break;
 	}
-	else if (pkg_id == 0x600A)
+	case 0x600A:
 	{
 		// set seed token
+		break;
+	}
+	case 0x600E:
+	{
+		// allocate buffer
+		const auto addr = vm::alloc(a1, vm::main);
+		if (!addr)
+			return CELL_ENOMEM;
+		vm::write32(a2, addr);
+		break;
+	}
+	case 0x600F:
+	{
+		// release buffer
+		if (!vm::dealloc(a2, vm::main))
```

Comment:
Never expose vm::dealloc directly, you must ensure that the origin of the allocation is from sys_ss.

Reply:
Thanks for notifying.
Fixed it up.